src/main/java/chess/domain/piece/Bishop.java

```diff
@@ -0,0 +1,46 @@
+package chess.domain.piece;
+
+import java.util.function.Consumer;
+
+import chess.domain.Color;
+import chess.domain.board.Position;
+
+public final class Bishop extends Piece {
+	private static final int SCORE = 3;
+
+	public Bishop(Color color) {
+		super(color);
+	}
+
+	@Override
+	public void move(Position beforePosition,
+		Position afterPosition,
+		Consumer<Piece> moveFunction) {
+		if (!canMove(beforePosition, afterPosition)) {
+			throw new IllegalArgumentException(INVALID_TARGET_POSITION_EXCEPTION);
+		}
+		moveFunction.accept(this);
+	}
```

Comment:
```suggestion
	@Override
	public void move(Position beforePosition, Position afterPosition, Consumer<Piece> moveFunction) {
		if (!canMove(beforePosition, afterPosition)) {
			throw new IllegalArgumentException(INVALID_TARGET_POSITION_EXCEPTION);
		}
		moveFunction.accept(this);
	}
```

이게 눈에 더 잘 들어오지 않나요? 😅